### DISCUSSION TITLE
2.8.3 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#4245] [讨论]
· 修复了以下 1 个 Bug
　　[严重] 部分玩家无法启动游戏：释放 Java Wrapper 失败（#4243，@clang3722）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.8.3'>2.8.3</a>.</em>

**Comment by GorgeousAhri** [comment]
I can't open my game after this update. And it can be run normally in the pervious version

**Reply by hejiehao** [reply]
Send Crash Log

**Reply by MoYuan-CN** [reply]
Open a new Issue and send your crash log.
